jest.config.js

```diff
@@ -2,15 +2,15 @@ module.exports = {
   collectCoverage: true,
   coverageThreshold: {
     global: {
-      branches: 100,
-      functions: 100,
-      lines: 100,
-      statements: 100,
```

Comment:
If this change was to fix non passing tests, let's try and fix the tests later so we get 100 coverage as the only thing I actually want to test is this package's API and all permutations of using the API. I don't want to have tests of implementation details.

Reply:
It was not my aim to fix test passing but 100% coverage is something that sometimes cannot be possible at all
A coverage of 85%-95% is totally valid

Lines like https://github.com/linterjs/core/blob/master/src/errors.ts#L4-L7 wont be covered (currently) 🤔